Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Lingmo OS/lingmo-settings](https://hosted.weblate.org/projects/lingmoos/lingmo-settings/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/lingmoos/lingmo-settings/horizontal-auto.svg)

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 93.8% (261 of 278 strings)

Translation: Lingmo OS/lingmo-settings
Translate-URL: https://hosted.weblate.org/projects/lingmoos/lingmo-settings/pt_BR/

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/LingmoOS/lingmo-settings?pullRequest=61) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/LingmoOS/lingmo-settings?pullRequest=61) before we can accept your contribution.<br/>**0** out of **10** committers have signed the CLA.<br/><br/>:x: weblate<br/>:x: AMITYEFET<br/>:x: SantosSi<br/>:x: atk8877<br/>:x: halitsever<br/>:x: arifpedia<br/>:x: Atalanttore<br/>:x: tarikomo<br/>:x: AntonioOliveira2<br/>:x: Filledipp<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/LingmoOS/lingmo-settings?pullRequest=61) it.</sub>

SantosSi committed:
Translated using Weblate (Portuguese (Portugal))

Currently translated at 95.6% (266 of 278 strings)

Translation: Lingmo OS/lingmo-settings
Translate-URL: https://hosted.weblate.org/projects/lingmoos/lingmo-settings/pt_PT/

Commit:
Translated using Weblate (Ukrainian)

Currently translated at 100.0% (278 of 278 strings)

Translation: Lingmo OS/lingmo-settings
Translate-URL: https://hosted.weblate.org/projects/lingmoos/lingmo-settings/uk/

AMITYEFET committed:
Translated using Weblate (Hebrew (Israel))

Currently translated at 100.0% (278 of 278 strings)

Translation: Lingmo OS/lingmo-settings
Translate-URL: https://hosted.weblate.org/projects/lingmoos/lingmo-settings/he_IL/

Commit:
Translated using Weblate (French)

Currently translated at 42.0% (117 of 278 strings)

Translation: Lingmo OS/lingmo-settings
Translate-URL: https://hosted.weblate.org/projects/lingmoos/lingmo-settings/fr/

Atalanttore committed:
Translated using Weblate (German)

Currently translated at 43.1% (120 of 278 strings)

Translation: Lingmo OS/lingmo-settings
Translate-URL: https://hosted.weblate.org/projects/lingmoos/lingmo-settings/de/

Commit:
Added translation using Weblate (Moldavian)

atk8877 committed:
Translated using Weblate (Arabic)

Currently translated at 84.5% (235 of 278 strings)

Translation: Lingmo OS/lingmo-settings
Translate-URL: https://hosted.weblate.org/projects/lingmoos/lingmo-settings/ar/

Commit:
Translated using Weblate (Russian)

Currently translated at 44.6% (124 of 278 strings)

Translation: Lingmo OS/lingmo-settings
Translate-URL: https://hosted.weblate.org/projects/lingmoos/lingmo-settings/ru/

Commit:
Translated using Weblate (Moldavian)

Currently translated at 15.8% (44 of 278 strings)

Translation: Lingmo OS/lingmo-settings
Translate-URL: https://hosted.weblate.org/projects/lingmoos/lingmo-settings/ro_MD/

Commit:
Translated using Weblate (Russian)

Currently translated at 45.6% (127 of 278 strings)

Translation: Lingmo OS/lingmo-settings
Translate-URL: https://hosted.weblate.org/projects/lingmoos/lingmo-settings/ru/

atk8877 committed:
Translated using Weblate (Arabic)

Currently translated at 100.0% (278 of 278 strings)

Translation: Lingmo OS/lingmo-settings
Translate-URL: https://hosted.weblate.org/projects/lingmoos/lingmo-settings/ar/

Commit:
Translated using Weblate (Moldavian)

Currently translated at 47.4% (132 of 278 strings)

Translation: Lingmo OS/lingmo-settings
Translate-URL: https://hosted.weblate.org/projects/lingmoos/lingmo-settings/ro_MD/

atk8877 committed:
Translated using Weblate (Arabic)

Currently translated at 100.0% (278 of 278 strings)

Translation: Lingmo OS/lingmo-settings
Translate-URL: https://hosted.weblate.org/projects/lingmoos/lingmo-settings/ar/

Commit:
Translated using Weblate (Moldavian)

Currently translated at 56.8% (158 of 278 strings)

Translation: Lingmo OS/lingmo-settings
Translate-URL: https://hosted.weblate.org/projects/lingmoos/lingmo-settings/ro_MD/

Commit:
Translated using Weblate (Moldavian)

Currently translated at 61.1% (170 of 278 strings)

Translation: Lingmo OS/lingmo-settings
Translate-URL: https://hosted.weblate.org/projects/lingmoos/lingmo-settings/ro_MD/

Commit:
Translated using Weblate (Moldavian)

Currently translated at 74.4% (207 of 278 strings)

Translation: Lingmo OS/lingmo-settings
Translate-URL: https://hosted.weblate.org/projects/lingmoos/lingmo-settings/ro_MD/

arifpedia committed:
Translated using Weblate (Indonesian)

Currently translated at 100.0% (278 of 278 strings)

Translation: Lingmo OS/lingmo-settings
Translate-URL: https://hosted.weblate.org/projects/lingmoos/lingmo-settings/id/

halitsever committed:
Translated using Weblate (Turkish)

Currently translated at 47.8% (133 of 278 strings)

Translation: Lingmo OS/lingmo-settings
Translate-URL: https://hosted.weblate.org/projects/lingmoos/lingmo-settings/tr/

Filledipp committed:
Translated using Weblate (Swedish)

Currently translated at 100.0% (278 of 278 strings)

Translation: Lingmo OS/lingmo-settings
Translate-URL: https://hosted.weblate.org/projects/lingmoos/lingmo-settings/sv/

AntonioOliveira2 committed:
Translated using Weblate (Portuguese (Portugal))

Currently translated at 100.0% (278 of 278 strings)

Translation: Lingmo OS/lingmo-settings
Translate-URL: https://hosted.weblate.org/projects/lingmoos/lingmo-settings/pt_PT/

tarikomo committed:
Translated using Weblate (Japanese)

Currently translated at 71.2% (198 of 278 strings)

Translation: Lingmo OS/lingmo-settings
Translate-URL: https://hosted.weblate.org/projects/lingmoos/lingmo-settings/ja/
